src/components/Clients/ClientListTableRow.jsx

```diff
@@ -48,13 +50,7 @@ const ClientListTableRow = ({ labelId, client, state, dispatch }) => {
 
   // Event handler for deleting client from client list
   const handleDeleteClient = async () => {
```

Comment:
This function is called from the new `DeleteModal` when a user clicks "Yes", but it doesn't close the modal.  I believe we need to add another line at the end of the function:
`setShowDeleteModal(false);`

Reply:
yes I should add a line at the end of the function setShowDeleteModal(false)